scripts/_common.sh

```diff
@@ -74,51 +56,6 @@ apply_layout() {
 # EXPERIMENTAL HELPERS
 #=================================================
 
-ynh_send_readme_to_admin() {
```

Comment:
The call to `ynh_send_readme_to_admin` in the install script should be replaced with a `POST_INSTALL.md` notification

Reply:
@oiseauroch would you like me to do it ?